wormhole-connect/src/views/v2/Bridge/AssetPicker/TokenWarnings.tsx

```diff
@@ -27,18 +27,18 @@ const useStyles = makeStyles()((theme: any) => ({
   },
   link: {
     textDecoration: 'underline',
-    opacity: '0.8',
+    opacity: 0.8,
```

Comment:
I've changed the opacity value to be between 0-1 (instead of a percentage) and a number (instead of string) across the redesign code for consistency. You will see more instances of these below.